openstudiocore/src/model/ScheduleTypeRegistry.cpp

```diff
@@ -251,8 +251,8 @@ ScheduleTypeRegistrySingleton::ScheduleTypeRegistrySingleton()
     {"ElectricEquipment","Electric Equipment","schedule",true,"",0.0,1.0},
     {"ElectricEquipmentITEAirCooled", "Design Power Input", "designPowerInputSchedule", true, "", 0.0, 1.0},
     {"ElectricEquipmentITEAirCooled", "CPU Loading", "cPULoadingSchedule", true, "", 0.0, 1.0},
-    {"ElectricEquipmentITEAirCooledDefinition", "Supply Temperature Difference", "supplyTemperatureDifferenceSchedule", true, "Temperature", 0.0, 50.0},
-    {"ElectricEquipmentITEAirCooledDefinition", "Return Temperature Difference", "returnTemperatureDifferenceSchedule", true, "Temperature", -50.0, 50.0},
+    {"ElectricEquipmentITEAirCooledDefinition", "Supply Temperature Difference Schedule", "supplyTemperatureDifferenceSchedule", true, "Temperature", 0.0, 50.0},
+    {"ElectricEquipmentITEAirCooledDefinition", "Return Temperature Difference Schedule", "returnTemperatureDifferenceSchedule", true, "Temperature", -50.0, 50.0},
```

Comment:
Confirmed this matches the `ElectricEquipmentITEAirCooledDefinition_Impl::getScheduleTypeKeys`:

https://github.com/NREL/OpenStudio/blob/a1449287fe7855d289d7dc80eb5aa474ef9de9f6/openstudiocore/src/model/ElectricEquipmentITEAirCooledDefinition.cpp#L101

https://github.com/NREL/OpenStudio/blob/a1449287fe7855d289d7dc80eb5aa474ef9de9f6/openstudiocore/src/model/ElectricEquipmentITEAirCooledDefinition.cpp#L105

Reply:
The `[0;50]` and `[-50;50]` bounds are unusual though. 

@KaiyuSun what are the source for these? Are these bounds taken from the E+ source code?

@kbenne: Would replacing the (+-) 50 by `OptionalDouble()` be a more reasonable approach?

Reply:
I updated the return temperature difference range to [-50;0], see the latest change. These limitations are actually restricting the +- of the temperature difference. For supply temperature difference (IT inlet T - AHU supply T), it should always be positive; For return, it should always be negative (AHU return T - IT outlet T).

Reply:
@jmarrec @kbenne Please advise if there is more common method of implementing this kind of limitation. Thank you!

Reply:
If this is just a sign thing, then OptionalDouble() is the way to go.

Reply:
For peace of mind, changed it to https://github.com/NREL/OpenStudio/pull/3807